openedx/core/djangoapps/content_tagging/rules.py

```diff
@@ -62,7 +196,7 @@ def can_change_object_tag_taxonomy(user: UserType, taxonomy: oel_tagging.Taxonom
     Taxonomy users can tag objects using tags from any taxonomy that they have permission to view. Only taxonomy admins
     can tag objects using tags from disabled taxonomies.
```

Comment:
> "Only taxonomy admins can tag objects using tags from disabled taxonomies."

I'm not sure about this.. I think "disabled" means it can't be used to tag objects, even if you're an admin.

Reply:
I think that it could make sense to let an admin clean tags from a disabled taxonomy. But we can simplify and let only enabled taxonomies be used.

What do you think?

Reply:
> "Only taxonomy admins can tag objects using tags from disabled taxonomies."

Yeah I agree with @pomegranited. I don't think we have any need for this. In fact for the MVP we won't really be using disabled taxonomies at all.

Either of these options is fine with me:
* If the taxonomy is disabled, nobody can modify its tags on an object.
* If the taxonomy is disabled, admins can _remove_ tags from objects, but not add new ones.

This only affects the API. In the UI, we will never display tags from disabled taxonomies, even to admins. (I think)

Reply:
Agreed!
I went to: "If the taxonomy is disabled, nobody can modify its tags on an object."